<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCC::KillSwitchDashboard</class>
 <widget class="QWidget" name="OCC::KillSwitchDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kill Switch Dashboard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Protection Statistics</string>
     </property>
     <property name="styleSheet">
      <string>font-size: 16px; font-weight: bold;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="countersGroup">
     <property name="title">
      <string>Overview</string>
     </property>
     <layout class="QHBoxLayout" name="countersLayout">
      <item>
       <widget class="QFrame" name="filesFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <layout class="QVBoxLayout" name="filesLayout">
         <item>
          <widget class="QLabel" name="filesAnalyzedLabel">
           <property name="text">
            <string>0</string>
           </property>
           <property name="styleSheet">
            <string>font-size: 24px; font-weight: bold; color: #2196F3;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="filesDescLabel">
           <property name="text">
            <string>Files Analyzed</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="threatsFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <layout class="QVBoxLayout" name="threatsLayout">
         <item>
          <widget class="QLabel" name="threatsBlockedLabel">
           <property name="text">
            <string>0</string>
           </property>
           <property name="styleSheet">
            <string>font-size: 24px; font-weight: bold; color: #FF5722;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="threatsDescLabel">
           <property name="text">
            <string>Threats Blocked</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="detectorsGroup">
     <property name="title">
      <string>Top Detectors</string>
     </property>
     <layout class="QVBoxLayout" name="detectorsLayout">
      <item>
       <widget class="QListWidget" name="detectorsListWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>QListWidget { font-family: monospace; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="timelineGroup">
     <property name="title">
      <string>Activity Timeline</string>
     </property>
     <layout class="QVBoxLayout" name="timelineLayout">
      <item>
       <layout class="QHBoxLayout" name="timelineHeaderLayout">
        <item>
         <widget class="QComboBox" name="timeRangeComboBox">
          <item>
           <property name="text">
            <string>Last 24 hours</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Last 7 days</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Last 30 days</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="timelineSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="timelineCountLabel">
          <property name="text">
           <string>0 threats</string>
          </property>
          <property name="styleSheet">
           <string>font-weight: bold;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="timelineListWidget">
        <property name="styleSheet">
         <string>QListWidget { font-family: monospace; font-size: 11px; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export CSV...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
